lib/Test/Nginx/Util.pm

```diff
@@ -636,6 +640,38 @@ sub error_log_data () {
     return \@lines;
 }
 
+sub check_prevblock_error_log($) {
+    my ($is_end) = @_;
+
+    my $block = $PrevBlock;
+
+    if (!$PrevBlock) {
+        return;
+    }
+
+    my $name = $block->name;
+
+    if (defined $block->shutdown_error_log) {
+        if (!$is_end && $UseHup) {
+            SKIP: {
+                Test::More::skip("$name - skip the shutdown_error_log"
+                                 . " tests under the HUP reload mode", 1);
```

Comment:
I'm afraid the constant number of tests to be skipped being `1` is simply wrong. The `shutdown_error_log` can employ multiple tests when its value is an array reference. Be very careful about the test plan. Better test these cases yourself.